client/package.json

```diff
@@ -32,6 +32,7 @@
     "html2canvas": "^1.0.0-rc.5",
     "jest": "^24.9.0",
     "mapbox-gl": "^1.12.0",
+    "material-ui-image": "^3.3.2",
```

Comment:
Since we're not using this anymore, please remove it.

Reply:
Latest capture. Updated naming and minor style changes. 

<img width="1253" alt="Screen Shot 2022-08-22 at 10 23 57 PM" src="https://user-images.githubusercontent.com/3441200/186076690-4d28b1e6-a1ad-4a70-88e8-51b7b3e94ac4.png">